src/index.html

```diff
@@ -10,37 +10,54 @@
         <h1>Leaderboard</h1>
     </header>
     <main>
-        <!-- Section that displays the scoreboard -->
+        <!-- sections that shows score board -->
         <section class="show-scores">
             <div class="show-head">
                 <h2 class="show-heading">Recent scores</h2>
                 <button id="scores-refresh" type="button">Refresh</button>
             </div>
             <ul class="scores-list">
```

Comment:
- [ ] [optional] @mj-e-boc Great job on your design however it would be better if you only displayed a few scores maybe the 10 latest scores, this will make your design better